charts/prow/temp/marque_deployment.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Yes you can set up dependent charts, but it gets a little messy.  Rather than do that, I just added a kubelego config map and deployment which is less messy than dependent charts.  Removed references to marque.